cluster_configuration_tests.py

```diff
@@ -43,11 +48,11 @@ def test_build(self):
             self.log.info('Grafana Web UI: http://%s:3000',
                           self.monitors.nodes[0].public_ip_address)
 
-    def test_use_public_dns_names(self):
+    def test_change_seed_address_to_hostname(self):
```

Comment:
This test does uses public DNS name, why do we need to call hostname ?

Reply:
@fruch which name do you suggest? is "test_change_seed_address_to_dns_name" ok?

Reply:
the pipeline for it is called `features-dns-cluster-5min.jenkinsfile` so it's a bit better